apps/portals/ampals/src/pages/HomePage.tsx

```diff
@@ -87,63 +87,24 @@ export default function HomePage() {
       <ALLALSSlat />
       <div className={'home-bg-dark'}>
         <SectionLayout
-          title={'Featured Datasets'}
+          title={'More Resources'}
           centerTitle
-          ContainerProps={{ className: 'home-spacer' }}
+          ContainerProps={{
+            sx: {
+              maxWidth: '100% !important',
```

Comment:
Since this is applied to a [MUI Container](https://mui.com/material-ui/api/container/), you can use the `maxWidth` prop directly
```suggestion
            maxWidth: false,
            sx: {
```